keyboards/idobao/montex/v1rgb/keymaps/via/config.h

```diff
@@ -0,0 +1,8 @@
+// Copyright 2022 peepeetee (@peepeetee)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#if defined(RGB_MATRIX_ENABLE)
+    #define VIA_QMK_RGBLIGHT_ENABLE
+#endif
```

Comment:
Won't be needed on develop.

Reply:
not sure i understand - i'm not posting to develop, this is a new kb

Reply:
nm - i saw it in https://github.com/qmk/qmk_firmware/blob/e7931289918221081cbe2a7ea5df27a5d86324db/quantum/via.c#L38
deleting file instead